edtLHD/src/main/java/fr/univtln/lhd/view/edt/EdtGrid.java

```diff
@@ -16,7 +16,7 @@
 public class EdtGrid extends Grid {
 
     public enum Days { LUNDI, MARDI, MERCREDI, JEUDI, VENDREDI }
-    private List<String> hours;
+    private final List<String> hours;
```

Comment:
not sure if it's a good idea...

Reply:
This is straight from develop. I have not done any changes to this file, @Desmondmies ?